### PR TITLE
Resolve: #47 add .env support

### PR DESCRIPTION
**Pull request by JennieJi**

### CODE REVIEW

File: start.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const path = require('path');
+const { exec, spawn } = require('child_process');
+
+const result = require('dotenv').config();
+const gatsby = require('gatsby-cli');
```
Comment (thchia) [inline start.js]: I might be missing something but is this complete?

**Comment by JennieJi** [comment]
ah yes. gatsby cli is not exporting anything but reading params from process

Best Regards,
Jennie Ji

________________________________

File: start.js (hunk 1 above)
Comment (wgao19) [reply start.js]: works for me, you may need to create an `.env` file and put one line inside

```
GITHUB_TOKEN=whateverthatis
```

**Comment by wgao19** [comment]
Thanks for the PR! Merging and closing #47

File: start.js (hunk 1 above)
Comment (thchia) [reply start.js]: According to Gatsby docs, you can just import dotenv in gatsby-config. Anyway we can revisit this in the future if necessary.

**Comment by JennieJi** [comment]
oh cool, doesn't know gatsby config support that, let me check and update later

Best Regards,
Jennie Ji

________________________________

**Comment by thchia** [comment]
[Source](https://www.gatsbyjs.org/docs/environment-variables/#server-side-nodejs)

File: start.js (hunk 1 above)
Comment (wgao19) [reply start.js]: wow, today i learned

**Pull request merged**